blocks/copilot/copilot.js

```diff
@@ -138,52 +179,55 @@ export default function decorate(block) {
       content: `Parse and return only the JSON representation for "${promptStr}" without anything else`,
     });
     console.log('[copilot] Call chat api to parse prompt to JSON');
-    const response = await callAzureChatCompletionAPI(conversation);
+    let response = await callAzureChatCompletionAPI(conversation);
     conversation.push(response);
     console.log(`[copilot] Parsed Prompt ${response.content}`);
     const respContent = JSON.parse(response.content);
     console.log(`[copilot] Prompt parsed to JSON ${JSON.stringify(respContent, null, 2)}`);
-	
-	let text_adjective = "";
-	if(respContent.text_adjective && respContent.text_adjective != null) {
-		
-		// Prepare synonym chat api
-		console.log('[copilot] Initiate Synonym chat with Azure to train it to get prescribed vocab');
-		const synonym_conversation = await initiateSynonymChat();
-		console.log('[copilot] Synonym Chat initiated');
-		console.log(synonym_conversation);
-		
-		// Call chat api
-		synonym_conversation.push({
-		  role: 'user',
-		  content: `"${respContent.text_adjective}"`,
-		});
-		console.log('[copilot] Call chat api to get synonym');
-		const response = await callAzureChatCompletionAPI(synonym_conversation);
-		console.log(response);
-		synonym_conversation.push(response);
-		console.log(`[copilot] Chat Response Synonym ${response.content}`);
-		
-		text_adjective = response.content;
-
-	}
-	
-	let image_adjective = "";
-	if(respContent.image_adjective && respContent.image_adjective != null) {
-		image_adjective = respContent.image_adjective;
-	}
-	
-	let image_tone = "";
-	if(respContent.image_tone && respContent.image_tone != null) {
-		image_tone = respContent.image_tone;
-	}
-		
-	let generationConfig = {
-		"text_adjective" : text_adjective, 
-		"image_adjective" : image_adjective, 
-		"image_tone" : image_tone};
-	
-    const ingridients = templates[respContent.template];
+
+    let text_adjective = 'default';
+    if (respContent.text_adjective && respContent.text_adjective != null) {
+      // Prepare synonym chat api
+      console.log('[copilot] Initiate Synonym chat with Azure to train it to get prescribed vocab');
+      const synonym_conversation = await initiateSynonymChat();
+      console.log('[copilot] Synonym Chat initiated');
+      console.log(synonym_conversation);
+
+      // Call chat api
+      synonym_conversation.push({
+        role: 'user',
+        content: `"${respContent.text_adjective}"`,
+      });
+      console.log('[copilot] Call chat api to get synonym');
+      response = await callAzureChatCompletionAPI(synonym_conversation);
+      console.log(response);
+      synonym_conversation.push(response);
+      console.log(`[copilot] Chat Response Synonym ${response.content}`);
+
+      text_adjective = response.content || text_adjective;
+      if (text_adjective.toLowerCase.includes('verbose')) {
+        text_adjective = 'verbose';
+      } else if (text_adjective.toLowerCase.includes('concise')) {
```

Comment:
prompt is to return `more` or `less`
so the condition to check for `verbose` or `concise` will never arise